Geometry/CommonTopologies/test/ValidateRadial.cc

```diff
@@ -40,7 +41,7 @@ ValidateRadial::ValidateRadial(const edm::ParameterSet& cfg)
       printOut_(cfg.getParameter<bool>("PrintOut")),
       posOnly_(cfg.getParameter<bool>("PosOnly")),
       tokTrackerGeometry_(esConsumes<TrackerGeometry, TrackerDigiGeometryRecord>()) {
-  std::cout << "I'm ALIVE" << std::endl;
```

Comment:
The test script is looking for the string "ALIVE" in the output, see https://github.com/cms-sw/cmssw/blob/47f67fce7aaeaf012b9671049643ad4e66d6ba92/Geometry/CommonTopologies/test/runTest.sh#L5

If you move to LogVerbatim the script must be updated accordingly